### ISSUE TITLE
lockup

### ISSUE THREAD

**Issue by artman325** [Intercoin/StakingContract#73]
```
when you stake and receive tokens
CommunityCoins get a gradual lockup for N days
Where N is set by factory owner
setLockupDays()
and lockupDays is public
lockupDays = 0 means dont add any gradual minimums
```

**Comment by artman325** [comment]
@EGreg 
for now CommunityCoins locked up depends of Staking pool when user staking tokens.

1. is it need to add ability to owner change lockupDays?
2. is t need to change current lockup from non-gradual to gradual ?

**Comment by EGreg** [comment]
1. factory owner can setLockupDays(N)  on a CommunityCoin which will affect all future mints of awarded in exchange for staking 
3. for this version let's make lockup always gradual

**Comment by artman325** [comment]
> 1. factory owner can setLockupDays(N)  on a CommunityCoin which will affect all future mints of awarded in exchange for staking
> 2. for this version let's make lockup always gradual

I said that ability has already present (when creation pool, we can specify duration).  So just need to do lockup gradual instead non-gradual, correct?

**Comment by EGreg** [comment]
Yes, sure. Gradual minimums for the CommunityCoins when they are awarded, including the bonuses I guess.
